commons/src/main/java/org/eclipse/kapua/commons/configuration/ServiceConfigPredicates.java

```diff
@@ -9,16 +9,11 @@
  * Contributors:
  *     Eurotech - initial API and implementation
  *******************************************************************************/
-package org.eclipse.kapua.service.user.internal;
+package org.eclipse.kapua.commons.configuration;
 
 import org.eclipse.kapua.model.KapuaUpdatableEntityPredicates;
 
-public interface UserPredicates extends KapuaUpdatableEntityPredicates {
+public interface ServiceConfigPredicates extends KapuaUpdatableEntityPredicates {
```

Comment:
NAME predicate has been removed but this was not changed to `extends` from `KapuaNamedEntityPredicates`